QA - create erasure coding test

YuliaKovalenko opened the pull request:
Created separate agents_function.js with create, delete and other actions for agents
erasure_coding_test contains steps:
1. create agents (number in arg)
2. verify create bucket
3. verify upload md5 file with random size 100k  or 1MB or 1GB
4. verify download md5 file
5. stop agent
6. check number of left online OPTIMAL agents 
7. verify download md5 file
8. start agent
9. check number of online agents 
10. check number of online OPTIMAL agents 
11. verify create bucket
12. verify upload md5 file with random size 100k  or 1MB or 1GB
13. verify download md5 file
14. delete agents

### Explain the changes:
- checking possibilities of rebuild agents and functionality after stopped one of agents

### Issues (fixed #xxxx / opened technical debt #yyyy / partial fix to #zzzz)
- 

### Reminders
- User Experience is top priority
- Design for failure
- Keep it simple
- Write for cross-platform
- Run `npm test`
- Create a unit-test - the first is the hardest
- Imagine the support call - Add diagnostics info, phonehome info, activity log events, external syslog
- Upgrade from previous version - DB schema, server platform, agents install, new packages added to deploymet

nimrod-becker reviewed (changes requested):
Hi Yulia, 
See my comments :)

Also, can we add the ability to get as parameters to the test totalAgents and NumFailedAgents

totalAgents would be the number of agents the tests starts (today you're doing it with a hard coded 3)
numFailedAgents would be the number of times we call to stopRandomAgent() (today you are doing it with 1).

So the flow is:
1. Create the entire test env. (agents according to the number totalAgents which is received as a parameter) 
2. Loop on specific file size, per each file size X:
  2.1 Upload a file in the size of X
  2.2 Read the file and verify md5
  2.3 stop numFailedAgents random agents
  2.4 immediately read and verify the file
  2.5 wait until rebuild was done (not by const timeout, check the status of the object)
  2.6 once rebuild was done read and verify the file
  2.7 start the previously failed agents
  2.8 wait till all agents are up and functional (again by looping and checking status)

Commit:
add erasure_coding_tests

add rebuild function, requested changes

change rpc connection type

waiting for 3 online replicas per part

test run

test without 1Gb

check chunks

Revert: not needed changes

liranmauda reviewed (changes requested):
1. please add comments
